### PR TITLE
Cohort column description

### PR DESCRIPTION
**Pull request by eanders**
## Description

Include a summary of the changes and a link to the related issue. List any dependencies that are required for this change.

## Type of change
- [ ] Bug fix
- [ ] New feature (adds functionality)
- [x] Code clean-up / housekeeping
- [ ] Dependency update

## Checklist before requesting review
- [ ] I have performed a self-review of my code
- [ ] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [ ] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [ ] My code follows the style guidelines of this project (rubocop)
- [ ] I have updated the documentation (or not applicable)
- [ ] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: app/models/cohort_columns/individual_in_most_recent_enrollment.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class IndividualInMostRecentEnrollment < ReadOnly
     attribute :column, String, lazy: true, default: :individual_in_most_recent_homeless_enrollment
     attribute :translation_key, String, lazy: true, default: 'Presented as Individual'
     attribute :title, String, lazy: true, default: ->(model, _attr) { Translation.translate(model.translation_key) }
-    attribute :description_translation_key, String, lazy: true, default: 'Client presented as an individual in the most recent homeless enrollment'
+    attribute :description_translation_key, String, lazy: true, default: 'Client presented as an individual (with no other household members) in the most recent homeless enrollment'
```
Comment (awisema) [inline app/models/cohort_columns/individual_in_most_recent_enrollment.rb]:
```suggestion
    attribute :description_translation_key, String, lazy: true, default: 'Client presented as an individual (i.e., with no other household members) in the most recent homeless enrollment'
```

File: app/models/cohort_columns/sober.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ class Sober < ReadOnly
     attribute :column, String, lazy: true, default: :sober
     attribute :translation_key, String, lazy: true, default: 'Appropriate for Sober Supportive Housing'
     attribute :title, String, lazy: true, default: ->(model, _attr) { Translation.translate(model.translation_key) }
-    attribute :description_translation_key, String, lazy: true, default: ->(model, _attr) { "#{model.translation_key} Description" }
+    attribute :description_translation_key, String, lazy: true, default: 'Client was marked appropriate for Sober Supportive Housing in tehir most recent TC HAT'
```
Comment (awisema) [inline app/models/cohort_columns/sober.rb]:
```suggestion
    attribute :description_translation_key, String, lazy: true, default: 'Client was marked appropriate for Sober Supportive Housing in their most recent TC HAT'
```

File: app/models/cohort_columns/vispdat_priority_score.rb [hunk 3]
```diff
@@ -9,7 +9,7 @@ class VispdatPriorityScore < ReadOnly
     attribute :column, String, lazy: true, default: :vispdat_priority_score
     attribute :translation_key, String, lazy: true, default: 'VI-SPDAT Priority Score'
     attribute :title, String, lazy: true, default: ->(model, _attr) { Translation.translate(model.translation_key) }
-    attribute :description_translation_key, String, lazy: true, default: ->(model, _attr) { "#{model.translation_key} Description" }
+    attribute :description_translation_key, String, lazy: true, default: 'VI-SPDAT score plus additonal prioritization logic.'
```
Comment (awisema) [inline app/models/cohort_columns/vispdat_priority_score.rb]:
```suggestion
    attribute :description_translation_key, String, lazy: true, default: 'VI-SPDAT score plus additional prioritization logic.'
```

**Review by awisema — APPROVED** [review]

File: app/models/cohort_columns/active_in_cas_match.rb [hunk 4]
```diff
@@ -9,7 +9,7 @@ class ActiveInCasMatch < ReadOnly
     attribute :column, String, lazy: true, default: :active_in_cas_match
     attribute :translation_key, String, lazy: true, default: 'Active CAS Match'
     attribute :title, String, lazy: true, default: ->(model, _attr) { Translation.translate(model.translation_key) }
-    attribute :description_translation_key, String, lazy: true, default: ->(model, _attr) { "#{model.translation_key} Description" }
+    attribute :description_translation_key, String, lazy: true, default: 'True if a client is involced in a match in CAS that is ongoing.'
```
Comment (dtgreiner) [inline app/models/cohort_columns/active_in_cas_match.rb]:
```suggestion
    attribute :description_translation_key, String, lazy: true, default: 'True if a client is involved in a match in CAS that is ongoing.'
```

File: app/models/cohort_columns/days_homeless_last_three_years.rb [hunk 5]
```diff
@@ -9,7 +9,7 @@ class DaysHomelessLastThreeYears < ReadOnly
     attribute :column, String, lazy: true, default: :days_homeless_last_three_years
     attribute :translation_key, String, lazy: true, default: 'Days Homeless in the last 3 years*'
     attribute :title, String, lazy: true, default: ->(model, _attr) { Translation.translate(model.translation_key) }
-    attribute :description_translation_key, String, lazy: true, default: ->(model, _attr) { "#{model.translation_key} Description" }
+    attribute :description_translation_key, String, lazy: true, default: 'Days spent in homeless projects that are not overlapping housed dates.  Calculated from HMIS data.'
```
Comment (dtgreiner) [inline app/models/cohort_columns/days_homeless_last_three_years.rb]:
```suggestion
    attribute :description_translation_key, String, lazy: true, default: 'Days spent in homeless projects that are not overlapping housed dates. Calculated from HMIS data.'
```

**Review by dtgreiner — APPROVED** [review]
The period usage at the end of the new descriptions feels inconsistent to me, but I may be missing some context.

**Pull request merged**